src/elements/map-container/map-container.visual.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { html } from 'lit';
+import { styleMap } from 'lit/directives/style-map.js';
+
+import { describeViewports, visualDiffDefault } from '../core/testing/private.js';
+
+import './map-container.js';
+import '../header.js';
+import '../title.js';
+
+describe(`sbb-map-container`, () => {
+  describeViewports({ viewports: ['zero', 'medium'], viewportHeight: 500 }, () => {
+    it(
+      visualDiffDefault.name,
+      visualDiffDefault.with(async (setup) => {
+        await setup.withFixture(
+          html`
+            <sbb-header expanded hide-on-scroll>
```

Comment:
Does it make a difference if hide-on-scroll is set or not?

Reply:
mmm no, it's a leftover from the tentative of testing the scroll behavior :/

Reply:
maybe you could remove it to not generate wrong assumptions, but not important